### DISCUSSION TITLE
Automatically filter spam comments on Gist

### DISCUSSION THREAD

**Discussion by captn3m0** [community/community#7945] [Other Feature Feedback, Questions, & Ideas]
A mildly popular gist of mine gets me one spam comment every few weeks. I don't mind the spam or the notifications (or cleaning them up from the Gist by reporting/deleting).

What I feel bad about is that the spam is terribly low quality, the kind that would have been caught by akismet spam filters on Wordpress a decade ago. Here's screenshots from the email notifications I got:

![image](https://user-images.githubusercontent.com/584253/142737029-d0e097f9-1729-4bf1-acd4-e14e6958e887.png "Screenshots of low-quality spam text on my gist")

![image](https://user-images.githubusercontent.com/584253/142737033-da113a52-2096-4c5f-be64-b54fc79aa855.png)

![image](https://user-images.githubusercontent.com/584253/142737036-824d8b31-af91-4c12-9bd1-75cee20e79a3.png)

![image](https://user-images.githubusercontent.com/584253/142737041-b7cdf892-a4ce-4a17-a823-6c86e9eacfc2.png)

![image](https://user-images.githubusercontent.com/584253/142737043-000657c3-24fb-46f8-a6a9-679a24d7c9bb.png)

![image](https://user-images.githubusercontent.com/584253/142737051-8b108db3-ddaa-44b4-bddb-817c1697ca11.png)

![image](https://user-images.githubusercontent.com/584253/142737058-49970335-8ece-4c1e-8c85-cb78d4b91e22.png)

When I get these notifications, it feels to me that there is no spam filtering on Gist comments (especially when the good:bad ratio is almost 1:10)

Request: Add _some basic spam filtering_ on comments on gists, or improve it to stop such low quality spam from going through.

**Comment by captn3m0** [comment]
Still keep getting spam on this gist.

**Reply by captn3m0** [reply]
Never-ending, low-tier spam.

![same gist, more spam screenshot](https://github.com/community/community/assets/584253/152e0521-3d38-48e2-9609-850fc62b8caa)

**Comment by captn3m0** [comment]
Still keep getting spam comments:

![screenshot of spam comments on the gist](https://user-images.githubusercontent.com/584253/178954568-8cc7d9e6-64e3-4119-aa14-1485eb5dd827.png)

**Comment by AustinWise** [comment]
Also the report comment button form always fails for me, at least for the one spam comment I got. Perhaps because the gist is 13 years old?

**Comment by fed-franz** [comment]
I'd like to add, that the spam report process is excessively long.
I recently found several spam comments on one of my gists. I can't waste 1 minute for each one of them...
This should be a matter of two clicks: "report->spam"

**Comment by spotype** [comment]
Response #1
